### PR TITLE
Use queued time, not created_at, for 1 day max retry [#178855164]

### PR DESCRIPTION
**Pull request by alaroia-cfa**

### CODE REVIEW

File: app/state_machines/efile_submission_state_machine.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ class EfileSubmissionStateMachine
       submission.transition_to!(:waiting) if transition.efile_errors.all?(&:auto_wait)
       if transition.efile_errors.all? { |efile_error| EfileError.error_codes_to_retry_once.include?(efile_error.code) }
         already_auto_resubmitted = submission.previously_transmitted_submission && submission.previously_transmitted_submission.efile_submission_transitions.where(to_state: :resubmitted
-          ).any? { |transition| puts(transition.metadata); transition.metadata.dig("auto_resubmitted") }
+          ).any? { |transition| transition.metadata.dig("auto_resubmitted") }
```
Comment (alaroia-cfa) [inline app/state_machines/efile_submission_state_machine.rb:87]: This change removes a `puts(...)` that I left in the code earlier by accident.

File: spec/models/efile_submission_spec.rb [hunk 2]
```diff
@@ -445,14 +445,15 @@
   end
 
   describe "#retry_send_submission" do
-    context "when the submission is fairly recent" do
-      before do
```
Comment (alaroia-cfa) [inline spec/models/efile_submission_spec.rb:449]: The `before do` is not really removed; it's moved higher-up into the `describe ...` out of each `context`.

File: app/state_machines/efile_submission_state_machine.rb [hunk 3]
```diff
@@ -84,7 +84,7 @@ class EfileSubmissionStateMachine
...
         unless already_auto_resubmitted
           submission.transition_to!(:resubmitted, {auto_resubmitted: true})
```
Comment (bytheway875) [inline app/state_machines/efile_submission_state_machine.rb:89]: incredibly picky, but im surprised that rubocop doesnt try to enforce spaces around the hash here.

**Review by bytheway875 — APPROVED** [review]

File: app/state_machines/efile_submission_state_machine.rb (hunk 3 above)
Comment (alaroia-cfa) [reply app/state_machines/efile_submission_state_machine.rb:89]: @bytheway875 Fascinating! Me too.

**Pull request merged**